Source/Public/Convert-JsonToBicep.ps1

```diff
@@ -0,0 +1,41 @@
+function Convert-JsonToBicep {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory, 
+            ParameterSetName = 'String')]
+        [ValidateNotNullOrEmpty()]
+        [ValidateScript({$_ | Convertfrom-Json})]
```

Comment:
Add a try / catch block to improve error message. 
```PowerShell
[ValidateScript({ 
  try { $_ | Convertfrom-Json }
  catch {throw 'not a valid json'}
})]
```

Error as it looks now:
```PowerShell
> Convert-JsonToBicep -String 'NotAJson'
Convert-JsonToBicep: Cannot validate argument on parameter 'String'. Conversion from JSON failed with error: Error parsing NaN value. Path '', line 1, position 1.
```

with catch:
```PowerShell
> Convert-JsonToBicep -String 'NotAJson'
Convert-JsonToBicep: Cannot validate argument on parameter 'String'. not a valid json
```